### PR TITLE
[IA-3323] Tidbits from Terra UI training

### PR DESCRIPTION
**Pull request by kyuksel**
This PR includes a couple of things we noticed while walking through code during the UI training. I forgot to PR them then. Better late than never :)

### CODE REVIEW

File: src/pages/library/Showcase.js [hunk 1]
```diff
@@ -84,7 +85,7 @@ const Showcase = () => {
   const [fullList, setFullList] = useState(stateHistory.featuredWorkspaces)
 
   useOnMount(() => {
-    const loadData = async () => {
+    const loadData = withErrorReporting('Error loading showcase', async () => {
```
Comment (nawatts) [inline src/pages/library/Showcase.js:88]: UX says these error messages should end with a period.

**Review by nawatts — APPROVED** [review]

File: src/pages/library/Showcase.js (hunk 1 above)
Comment (kyuksel) [reply src/pages/library/Showcase.js:88]: I posted on [Slack](https://broadinstitute.slack.com/archives/C34GAKQ5R/p1649855343820179?thread_ts=1648570756.312039&cid=C34GAKQ5R) to double-check with UX. We seem to have a ton of these messages with no full stop.

Comment (nawatts) [reply src/pages/library/Showcase.js:88]: I thought the outcome of the last [Slack discussion](https://broadinstitute.slack.com/archives/C34GAKQ5R/p1648570756312039) on punctuation was that these messages should end with a period.

Comment (kyuksel) [reply src/pages/library/Showcase.js:88]: I've had a difficult time being sure myself.

**Review by cahrens — APPROVED** [review]

**Pull request merged**